meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-machine-id.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Machine-id Mount Point
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=systemd-machine-id-commit.service
+
+[Service]
```

Comment:
In my opinion this would make the implementation less hacky. Cause it has that smell right now. @petrosagg any preference?